views/index.erb

```diff
@@ -0,0 +1,22 @@
+
+<% @messages.each do |message| %>
+  <ul style="list-style-type:none;">
+    <li> 
+      <h2><%= message.message%></h2>
+      Posted at <%= message.date[0...-10]%>
```

Comment:
perhaps you could add a method to Message called something like `pretty_date` which takes care of the formatting.  Or, even better, you could create a DatePresenter class, with a `pretty_date` that takes a raw date and then returns a pretty one.